### PR TITLE
#168 - Align Modal to Center of PDF if Signed

### PR DESCRIPTION
**Pull request by rishabjasrotia**
#168 - Align Modal to Center of PDF if Signed

Output:
image

### CODE REVIEW

**Review by raktima-opensignlabs — DISMISSED** [review]

File: microfrontends/SignDocuments/src/Component/SignYourselfPdf.js [hunk 1]
```diff
@@ -1036,7 +1048,10 @@ function SignYourSelf() {
           >
             {/* this modal is used show this document is already sign */}
 
-            <Modal show={showAlreadySignDoc.status}>
+            <Modal 
+              show={showAlreadySignDoc.status}
+              onShow={() => modalAlign()}
+            >
               <ModalHeader style={{ background: themeColor() }}>
```
Comment (raktima-opensignlabs) [inline microfrontends/SignDocuments/src/Component/SignYourselfPdf.js:1055]: There should be background color rgba(0, 0, 0, 0.75) as mentioned in #168  issue .

Comment (rishabjasrotia) [reply microfrontends/SignDocuments/src/Component/SignYourselfPdf.js:1056]: Hello @raktima-opensignlabs background backdrop added for modal.
image

File: microfrontends/SignDocuments/package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "react-scrollbars-custom": "^4.1.1",
     "react-signature-canvas": "^1.0.6",
     "reactour": "^1.19.1",
+    "select-dom": "^9.0.0",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
```
Comment (raktima-opensignlabs) [inline microfrontends/SignDocuments/package.json:38]: Is there a specific reason for using extra select-dom package.

Comment (rishabjasrotia) [reply microfrontends/SignDocuments/package.json:38]: Only as an helper class with lightweight selection.

Comment (raktima-opensignlabs) [reply microfrontends/SignDocuments/package.json:38]: Hello, @rishabjasrotia, can you please help us out to resolve conflict?

Comment (rishabjasrotia) [reply microfrontends/SignDocuments/package.json:38]: Yes sure I will resolve conflicts

**Comment by raktima-opensignlabs** [comment]
@nxglabs can you please deploy PR #188 on vercel.

**Pull request merged**